### DISCUSSION TITLE
Integrare vecchi sensori magnetici a filo in Home Assistant

### DISCUSSION THREAD

**Discussion by Andrea0776** [indomus/forum#3166] [GENERALE - Discussioni generiche]
salve a tutti
sto muovendo i primi passi in Home Assistant/Zigbee.
Chiedo scusa se i termini che uso non sono propriamente tecnici e, magari, vi faranno anche sorridere.

Sto studiando come creare, nell'appartamento che ho comprato recentemente, un sistema di allarme basato su HA e Zigbee (quantomeno questa mi sembra l'accoppiata più idonea).
I vecchi proprietari dell'appartamento avevano un vecchio sistema d'allarme cablato ed una centralina bella vecchiotta (del 1988).

Quello che vorrei riutilizzare, sono i sensori magnetici di apertura finestre, che sono stati installati, tramite degli appositi buchi, negli infissi (recenti e di ottima qualità) senza doverne montare di nuovi.
I sensori presenti sono a filo.

Quindi vi chiedo: esiste un qualche dispositivo Zigbee (visto che, se non sbaglio, sono quelli che consumano meno) ,a batteria, al quale collegare i due fili del sensore magnetico e che ne trasmetta lo stato al server centrale di HA/Zigbee ?

La cosa che ho trovato in rete, che più si avvicina a ciò che voglio realizzare io, è questa:
https://www.youtube.com/watch?v=V5xf22_aUTs
in pratica viene modificato un nuovo sensore magnetico per usarne la parte trasmissiva... però mi sembra un po' uno spreco.

Conoscete qualcosa che possa fare al caso mio ?
Grazie mille.
Andrea

**Comment by HA-n00b-alt** [comment]
Ho speso qualche ora su questo stesso problema in questi giorni. La cosa più interessante che ho trovato è konnected.io, se trovi altro e lo posti qua mi fai un favore :)

**Comment by indomus** [comment]
> Conoscete qualcosa che possa fare al caso mio ? Grazie mille. Andrea

In realtà è molto più facile di quanto credi. Ti basta adottate [ESPHome](https://indomus.it/componenti/esphome/) su un qualsiasi microcontroller ESP8266/ESP32 (costano cento lire) sul quale attestare i capi che arrivano dai sensori.

**Reply by HA-n00b-alt** [reply]
Grazie! Ho scoperto ieri che forse riesco a tenere l’allarme: è un Tecnoalarm T8-88 che *dovrebbe* avere la possibilità di attivare un protocollo proprietario (TechnoOUT) su IP. Se me lo confermano proverò a scrivere un plugin per HA, oppure posso comprare un gateway TecnoOUT<>MQTT (ho visto che Hsyco ne ha uno, anche se un po’ costoso…).

**Reply by Andrea0776** [reply]
> In realtà è molto più facile di quanto credi. Ti basta adottate [ESPHome](https://indomus.it/componenti/esphome/) su un qualsiasi >microcontroller ESP8266/ESP32 (costano cento lire) sul quale attestare i capi che arrivano dai sensori.

Grazie mille !
Ho comprato tre schedine ESP8266 (10€) ed effettivamente i primi test hanno dato esito positivo.
Ancora grazie !
